docs/build/integrations/indexers/sentio.md

```diff
@@ -0,0 +1,269 @@
+- Restart: Restart from the beginning.
+
+**Inspect Variables**
+
+The debugger automatically shows the local variables within the call context, and all the contract variables.
+
+<Figure src={require('./img/sentio14.png').default } width="100%" />
+
+The debugger also supports adding **user defined watched variables (similar to a regular debugger.)**
+
+<Figure src={require('./img/sentio15.png').default } width="100%" />
+
+**Limitations**
+
+- Contracts compiled with viaIR option are not fully supported.
+- When debugging a release build, since they are fully optimized, source-mapping issues and unexpected execution orders may present themselves.
+- When debugging a debug build, gas usage is ignored, which may cause different code execution. e.g. if the original transaction runs out of gas, the debug build will indicate the transaction fully executes.
+
+### Function-only Mode
+
+If single-step mode is turned off, the debugger will behave at the *function* level.
+
+**Use the debugger**
+
+The debugger has standard definitions of:
+
+- Next: proceeds to the next function call (depth first search order)
+- Previous: reverts to the previous function call
+- Step Over: proceeds to the next function call (**does not** follow nested calls)
+- Step Up: goes up one level
+
+**Inspect the variables**
+
+In this mode, developers can visualize **Inputs**, **Return Value** and **Gas info.**
+
+<Figure src={require('./img/sentio16.png').default } width="100%" />
+
+## Simulation
+
+The Sentio simulator allows you to run simulations and analyze the data collected in great detail.
+You can quickly begin simulations through the Sentio [UI](https://docs.sentio.xyz/sentio-debugger/simulation/simulation-ui) or by calling the [API](https://docs.sentio.xyz/sentio-debugger/simulation/simulation-api).
+
+### Simulation UI
+
+**From existing transaction**
+The simplest way to start a simulation is to click the simulator button as shown below, on a transaction that has been opened. 
+
+<Figure src={require('./img/sentio17.png').default } width="100%" />
+
+In this case, it will copy all the parameters from the existing transaction and you could make adjustments on top of it. Like block number, block index, gas fee, block header, state, etc.
+
+<Figure src={require('./img/sentio18.png').default } width="50%" />
+
+Click the simulate transaction button will save this run to the simulation history of your project and show you the result, just like what you see from the normal debugger UI.
+
+**Direct Build**
+
+You can also click the simulator button on the left navigation bar and go to the simulator page which shows all the history simulations. Click the simulation button on the right corner will pop a similar UI but without prepopulated transaction data.
+
+<Figure src={require('./img/sentio19.png').default } width="85%" />
+
+**Override Contract**
+
+Use the compilations tab to upload a local contract compilation, folder.
```

Comment:
Is the comma extra or maybe there is some missing words?

Reply:
nice catch! thanks!